26-dracula-ant/technical_specification.md

```diff
@@ -55,38 +57,59 @@ Deletion of work packages should be logged or implemented as deactivation (immed
 
 The work package access token is a random string generated by the Work Package Service on a POST request. Its SHA256 hash is stored in the database. The token itself is not stored, but it is encrypted with the `public_key` and then returned in the response.
 
-The frontend should display the work package ID together with the encrypted work package access token separated by a colon as a single string that can be copied and pasted to the CLI client, so that the CLI client gets both pieces of information.
+The frontend should display the work package ID together with the encrypted work package access token separated by a colon as a single string that can be copied and pasted to the CLI client (GHGA connector), so that the CLI client gets both pieces of information.
 
 ### Work order token
 
 This is a JWT with a very short time to live (maximum 30 seconds). It is signed by the Work Package Service and then encrypted with the `public_key` of the user, similar to the work package access token. It contains the following claims:
 
 - `type`: `download` or `upload`
 - `file_id`: the ID of the file that shall be downloaded or uploaded
+- `file_ext`: the file extension of the requested filed including the dot
 - `user_id`: the internal ID of the user
-- `public_key`: the public key stored in the work package
+- `public_key`: the public key of the user stored in the work package
```

Comment:
```suggestion
- user_public_crypt4gh_key`: the public key of the user stored in the work package
```

maybe rename this for consistenxy with the work package

Reply:
The idea was to keep the work order token small (it's passed in the header). Usually JWT claim names are very short.

But yes, consistency is probably more important. I renamed it now.